hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/AbstractFindTargetGreedy.java

```diff
@@ -228,6 +228,9 @@ public void increaseSizeEntering(DatanodeDetails target, long size) {
       if (totalEnteringSize < config.getMaxSizeEnteringTarget()) {
         //reorder
         potentialTargets.add(nodeManager.getUsageInfo(target));
+      } else {
+        logger.debug("Datanode {} removed from the list of potential targets. The total size of data entering it in " +
```

Comment:
You can add LOGGER.isDebugEnabled() before logging

Reply:
There is no need to wrap it in an IF statement unless there is expensive logic to form the message. Inside the logger it has the isDebugEnabled check already.

Reply:
Ya I see it's there inside as well so not required.